packages/vite/src/node/plugin.ts

```diff
@@ -323,6 +323,20 @@ export interface Plugin<A = any> extends RollupPlugin<A> {
   >
 }
 
+export interface CustomPluginOptionsVite {
+  /**
+   * If this is a CSS Rollup module, you can scope to its importer's exports
+   * so that if those exports are treeshaken away, the CSS module will also
+   * be treeshaken.
+   *
+   * Example config if the CSS id is `/src/App.vue?vue&type=style&lang.css`:
+   * ```js
+   * cssScopeTo: ['/src/App.vue', 'default']
+   * ```
+   */
+  cssScopeTo?: [string, string | undefined]
```

Comment:
I changed the interface to the one I felt easier to implement by proposal for now.
@bluwy What was the reason of your interface? I'm wondering the usage of allowing multiple files here.

Reply:
I don't have a concrete usecase. Was thinking that maybe there's some sort of unocss-like tool that generates scoped css derived from multiple modules somehow that could benefit from this. So I futureproofed it as a Record.